serve/graph/model/utils.go

```diff
@@ -0,0 +1,113 @@
+package model
```

Comment:
No utils files, please. They end up being kitchen-sinks, where everything can be.

Reply:
I removed the utils file and moved the deref function to resolver.